### PR TITLE
Replace `Card` with `FrontCard` after merge issue

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Fixes an issue that got introduced after I merged #5383 

## Why?
My merge really should have thrown a merge conflict but for some reason it breezed through and started throwing errors on `main`

### CODE REVIEW

File: dotcom-rendering/src/web/components/DynamicPackage.tsx [hunk 1]
```diff
@@ -120,7 +68,6 @@ export const DynamicPackage = ({
 						trail={primary}
 						containerPalette={containerPalette}
 						showAge={showAge}
-						containerType="dynamic/package"
```
Comment (AshCorr) [inline dotcom-rendering/src/web/components/DynamicPackage.tsx:123]: Was this meant to be removed?

**Review by AshCorr — APPROVED** [review]

File: dotcom-rendering/src/web/components/DynamicPackage.tsx [hunk 2]
```diff
@@ -71,39 +46,12 @@ export const DynamicPackage = ({
...
-								<Card
+								<FrontCard
+									trail={card}
 									containerPalette={containerPalette}
-									containerType="dynamic/package"
 									showAge={showAge}
-									linkTo={card.url}
-									format={card.format}
-									headlineText={card.headline}
 									headlineSize="medium"
```
Comment (AshCorr) [inline dotcom-rendering/src/web/components/DynamicPackage.tsx:53]: Medium is the default size I think, shouldn't need to specify it.

Comment (oliverlloyd) [reply dotcom-rendering/src/web/components/DynamicPackage.tsx:53]: Ah, cool. Thanks, I'll update

File: dotcom-rendering/src/web/components/DynamicPackage.tsx (hunk 1 above)
Comment (oliverlloyd) [reply dotcom-rendering/src/web/components/DynamicPackage.tsx:123]: No. Thanks for spotting this!

**Pull request merged**